daphne/http_protocol.py

```diff
@@ -250,9 +250,8 @@ def handle_reply(self, message):
             # Write headers
             for header, value in message.get("headers", {}):
                 self.responseHeaders.addRawHeader(header, value)
-            
             if self.server.server_name and not self.responseHeaders.hasHeader("server"):
-                self.setHeader(b"server", self.server.server_name.lower().encode("utf-8"))
+                self.setHeader(b"server", self.server.server_name.encode())
```

Comment:
In this line to be exact. It should be lowered. Because asgi spec implies that headers should always be lowercase.

Reply:
Scratch this